tests/test_webhook.py

```diff
@@ -157,7 +158,7 @@ def test_finished_webhook_retries(self):
         """
         tm = Template(json_dumps(pc))
         rv = self.server.post(
-            URL_PREFIX + "/locations/nc_spm_08/processing_async_export",
+            URL_PREFIX + "/projects/nc_spm_08/processing_async_export",
```

Comment:
Same here - Maybe add test for both until GRASS GIS 9 ?

Reply:
Ist schon geändert. War leider noch nicht commited, da ich die Tests verschoben hatte